services/src/main/java/org/keycloak/authentication/authenticators/x509/AbstractX509ClientCertificateAuthenticatorFactory.java

```diff
@@ -201,6 +206,22 @@
         extendedKeyUsage.setLabel("Validate Extended Key Usage");
         extendedKeyUsage.setHelpText("Validates the extended purposes of the certificate's key using certificate's Extended Key Usage extension. Leaving the field blank will disable Extended Key Usage validation. See RFC 5280 for a detailed definition of X509 Extended Key Usage extension.");
 
+        ProviderConfigProperty certificatePolicy = new ProviderConfigProperty();
+        certificatePolicy.setType(STRING_TYPE);
+        certificatePolicy.setName(CERTIFICATE_POLICY);
+        certificatePolicy.setLabel("Validate Certificate Policy");
+        certificatePolicy.setHelpText("Validates the certificate policies of the certificate's key using certificate's Policy extension. Leaving the field blank will disable Certificate Policies validation. See RFC 5280 for a detailed definition of X509 Certificate Policy extension.");
```

Comment:
Perhaps we should mention here the possibility to provide a comma-separated list of OIDs?

Reply:
Sure! I will add "Multiple policies should be separated using a comma.", just like in the documentation PR.